src/library_html5.js

```diff
@@ -2080,6 +2080,7 @@ var LibraryJSEvents = {
       var touchEvent = JSEvents.touchEvent;
 #endif
       var idx = touchEvent>>2; // Pre-shift the ptr to index to HEAP32 to save code size
+      {{{ makeSetValue('touchEvent', C_STRUCTS.EmscriptenTouchEvent.timestamp, 'e.timeStamp', 'double') }}};
```

Comment:
```suggestion
      {{{ makeSetValue('touchEvent', C_STRUCTS.EmscriptenTouchEvent.timestamp, 'e.timeStamp', 'double') }}};
      var idx = touchEvent>>2; // Pre-shift the ptr to index to HEAP32 to save code size
```
Reversing the order keeps `idx` next to its uses.